ldms/src/store/darshan/darshan_stream_store.c

```diff
@@ -176,7 +176,7 @@ enum attr_ids {
        OFFSET_ID,
        LENGTH_ID,
        DURATION_ID,
-       TIMESTAMP_ID = 0,
+       TIMESTAMP_ID,
```

Comment:
How has this been tested? Since the timestamp field overlapped the job_id field, how could this ever have worked?

Reply:
What do you mean by the timestamp overlap the job _id field?

Reply:
@tom95858 What do you mean by the timestamp overlap the job _id field?